src/reducers/versions.spec.tsx

```diff
@@ -178,6 +180,67 @@ describe(__filename, () => {
       ]);
     });
 
+    it('adds all paths to expandedPaths when expandTree is dispatched', () => {
+      const path1 = 'path/1';
+      const path2 = 'path/2';
+      const entries = {
+        [path1]: createFakeEntry('directory', path1),
+        [path2]: createFakeEntry('directory', path2),
+      };
+
+      const version = { ...fakeVersion, file: { ...fakeVersionFile, entries } };
+      let state = reducer(undefined, actions.loadVersionInfo({ version }));
+
+      state = reducer(state, actions.expandTree({ versionId: version.id }));
+
+      expect(state).toHaveProperty(`versionInfo.${version.id}.expandedPaths`, [
+        path1,
+        path2,
+        getRootPath(createInternalVersion(version)),
+      ]);
+    });
+
+    it('does not add paths for files to expandedPaths when expandTree is dispatched', () => {
+      const path1 = 'path/1';
```

Comment:
This test would be easier to read if you used realistic examples, maybe like `scripts/` and `scripts/background.js`. Even though I understood the spec, I was having a hard time deciphering `path1` and `path2`.